src/main/java/com/commercetools/sync/producttypes/ProductTypeSync.java

```diff
@@ -280,7 +306,8 @@ private void handleError(@Nonnull final String errorMessage, @Nullable final Thr
                             final String errorMessage =
                                 format(CTP_PRODUCT_TYPE_UPDATE_FAILED, newProductType.getKey(),
                                     sphereException.getMessage());
-                            handleError(errorMessage, sphereException, 1);
+                            handleError(errorMessage, sphereException, 1, oldProductType, newProductType,
```

Comment:
sphereException -> exception?

Reply:
Will change it.

Reply:
Fixed: https://github.com/commercetools/commercetools-sync-java/pull/367/commits/70ca3bafc4e4a6a74e187f87cdcf2c38f3dfecf5